Красоты навёл и другие фиксы

savichev-igor opened the pull request:
:<

Commit:
fix(): big fixes

Commit:
fix(): big fixes

butch-pulp-fiction commented:
Зед: Эни-бени, трали-вали, с @PolinaKoval, @Anna1004 and @v-samoylov в прятки мы играли.
В грунт по уши закопали. Эни-бени, мени-мать, будут там теперь вонять.

Commit:
fix(): hot fix

Commit:
add search (#114)

* fix(pages/createQuests.js) fix redirect

* feat() add search

* # Conflicts:
#	controllers/pages.js
#	controllers/users.js
#	models/quests.js
#	pages/index/index.js
#	pages/quest/quest.hbs

* Merge branch 'master' of https://github.com/urfu-2015/team4 into search

# Conflicts:
#	models/quests.js
#	pages/index/index.js

* fix() remove comments

Commit:
create quest fix (#115)

* fix(pages/createQuests.js) fix redirect

* fix() create quest

* # Conflicts:
#	blocks/yandexMap/yandexMap.hbs
#	controllers/pages.js
#	models/quests.js
#	pages/createQuest/createQuest.css
#	pages/createQuest/createQuest.hbs

* fix() fix checkin

* # Conflicts:
#	models/quests.js

* # Conflicts:
#	models/quests.js

* fix

Commit:
fix(): hot fix

Commit:
Merge remote-tracking branch 'origin/Auth/Reg_formFixes' into Auth/Reg_formFixes

# Conflicts:
#	blocks/header/header.css
#	blocks/header/header.hbs
#	controllers/pages.js
